src/gui/dialogs/unit_recruit.hpp

```diff
@@ -47,7 +49,9 @@ class unit_recruit : public modal_dialog
 
 	void show_help();
 
-	std::vector<const unit_type*>& recruit_list_;
+	/// A vector of unit types in the order listed. Used by unit_recruit::post_show.
```

Comment:
"in the order listed in the UI." would be clearer to read.

Reply:
Done